samples/extensions/extended_dynamic_state2/extended_dynamic_state2.cpp

```diff
@@ -534,8 +530,10 @@ void ExtendedDynamicState2::build_command_buffers()
  */
 void ExtendedDynamicState2::create_descriptor_pool()
 {
+	const uint32_t frame_number = static_cast<uint32_t>(draw_cmd_buffers.size());
+
 	std::vector<VkDescriptorPoolSize> pool_sizes = {
-	    vkb::initializers::descriptor_pool_size(VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER, 3),
+	    vkb::initializers::descriptor_pool_size(VK_DESCRIPTOR_TYPE_UNIFORM_BUFFER, frame_number * 3),
```

Comment:
Why does the number of frames only affect the uniform buffer?

Reply:
changed to reflect allocation